api/src/main/java/bio/terra/tanagra/indexing/Main.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.tanagra.indexing;
+
+import bio.terra.tanagra.underlay.Entity;
+import bio.terra.tanagra.utils.FileUtils;
+import java.nio.file.Path;
+
+public final class Main {
+  private Main() {}
+
+  /**
+   * Main entrypoint for running indexing.
+   *
+   * <p>- Expand the user-specified underlay config:
+   *
+   * <p>./gradlew api:index -Dexec.args="EXPAND_CONFIG underlay.json output_dir"
+   *
+   * <p>- Kick off the indexing jobs for a single entity:
+   *
+   * <p>[dry run] ./gradlew api:index -Dexec.args="INDEX_ENTITY output_dir/underlay.json person
+   * DRY_RUN"
+   *
+   * <p>[actual run] ./gradlew api:index -Dexec.args="INDEX_ENTITY output_dir/underlay.json person"
+   */
+  public static void main(String... args) throws Exception {
+    // TODO: Consider using the picocli library for command parsing and packaging this as an actual
```

Comment:
The `./gradlew` command is not very pretty syntax to look at, but there are other drawbacks to using Gradle commands here:
- We potentially recompile the code for every command. 
- You have to sync the code to run indexing jobs.
Brian F mentioned one of the pain points with AoU CB is that you can't work on the code while you're running indexing, because you need to keep it at a current working version. Packaging a CLI is one way to avoid that. If we decide to do this with service endpoints instead, that would also solve those problems.